js/dom.js

```diff
@@ -11,9 +11,184 @@ function switchMenu() {
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  fullDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  image: {
+    src: './images/multipost.png',
+    alt: 'Project image',
+  },
+  technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  linkLive: '#',
+  linkSource: '#',
+};
+
+const multipost = document.querySelector('#multipost');
+const img = document.createElement('img');
+img.setAttribute('src', mainWork.image.src);
+img.setAttribute('alt', mainWork.image.alt);
+multipost.appendChild(img);
+
+const divText = document.createElement('div');
+divText.setAttribute('id', 'multipost-text');
+divText.classList.add('flex-column');
+multipost.appendChild(divText);
+
+function addMainContent(container, work) {
+  const title = document.createElement('h3');
+  title.textContent = work.name;
+  container.appendChild(title);
+
+  const p = document.createElement('p');
+  p.textContent = work.description;
+  container.appendChild(p);
+
+  const ul = document.createElement('ul');
+  ul.classList.add('flex-row');
+
+  work.technologies.forEach((item) => {
+    const li = document.createElement('li');
+    li.textContent = item;
+    ul.appendChild(li);
+  });
+  container.appendChild(ul);
+}
+
+function addButton(container, work) {
+  const newButton = document.createElement('button');
+  newButton.setAttribute('type', 'button');
+  newButton.setAttribute('id', `btn-${work.id}`);
+  newButton.textContent = 'See Project';
+  container.appendChild(newButton);
+}
+
+addMainContent(divText, mainWork);
+addButton(divText, mainWork);
+
+const grid = document.querySelector('#works-grid');
+
+const works = [];
+works.push(mainWork);
+
+for (let i = 1; i <= 6; i += 1) {
+  const work = {
+    id: i,
+    name: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    fullDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    image: {
+      src: './images/project-image.png',
+      alt: 'Project image',
+    },
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    linkLive: '#',
+    linkSource: '#',
+  };
+  works.push(work);
```

Comment:
- [ ] Since all projects won't be the same i.e one project will be about a Javascript game and another about an RoR app, it is better to have an array of objects listing all the project details within an object. So, duplicate the project you are having and then iterate through the array of objects just the same way you are doing.

Reply:
Sorry, I don't understand what I have to do differently here. I already have an array of objects, since the const works is an array that contains all the projects. Could you give me more details on how to do it? Thank you :grinning: